### PR TITLE
Account settings with phone/email storybook

### PR DESCRIPTION
**Pull request by buoyad**
image

Also:
* Fix actions addon in native storybook
* Add `decoration` prop to menu items for stuff that goes to the right of the title

Part 2 will be "confirm remove email/phone" dialog

- [ ] Storyshots before merge

r? @keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/floating-menu/menu-layout/index.native.js [hunk 1]
```diff
@@ -31,12 +31,14 @@ const MenuRow = (props: MenuRowProps) => (
     {props.view || (
       <>
         <Box2 direction="horizontal" fullWidth={true} centerChildren={true}>
+          {props.decoration && <Box style={styles.flexOne} />}
```
Comment (buoyad) [inline shared/common-adapters/floating-menu/menu-layout/index.native.js]: this is a nice solution for having a centered item and something on the side without having to set `width`s. A box with `flex: 1` on both sides will make sure the center item stays centered.

File: shared/settings/account/index.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+const AccountSettings = (props: Props) => {
+  let passwordLabel = props.hasPassword ? 'Change password' : 'Set a password'
+  if (props.hasPassword && Styles.isMobile) {
+    passwordLabel = 'Change'
+  }
+  return (
+    <Kb.ScrollView>
+      <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true}>
+        {/* Email + Phone */}
```
Comment (chrisnojima) [inline shared/settings/account/index.js]: if its worth commenting then i think its worth breaking out into its own component in this file

File: shared/settings/account/index.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  hasPassword: boolean,
+  onAddEmail: () => void,
+  onAddPhone: () => void,
+  onDeleteAccount: () => void,
+  onSetPassword: () => void,
+|}
+
+const AccountSettings = (props: Props) => {
+  let passwordLabel = props.hasPassword ? 'Change password' : 'Set a password'
+  if (props.hasPassword && Styles.isMobile) {
```
Comment (chrisnojima) [inline shared/settings/account/index.js]: 🐑 
```
const passwordLabel = props.hasPassword ? `Change${Styles.isMobile ? '' : ' password'}` : 'Set a password'
```
or
```
let passwordLabel
if (props.hasPassword) {
  passwordLabel = Styles.isMobile ? 'Change' : 'Change password'
} else {
 passwordLabel = 'Set a password'
}
```

i find logic to choose and then logic to overwrite more confusing

File: shared/settings/account/email-phone-row.js [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    top: -2,
+  },
+  badgeMenuItem: {
+    alignSelf: 'center',
+    marginLeft: 'auto',
+  },
+  container: {
+    height: Styles.isMobile ? 48 : 40,
+  },
+  flexOne: {
```
Comment (chrisnojima) [inline shared/settings/account/email-phone-row.js]: this should go into globalStyles

**Comment by buoyad** [comment]
@chrisnojima updated PTAL

**Pull request merged**